packages/vscode-ui5-language-assistant/package.json

```diff
@@ -24,7 +24,19 @@
             "verbose"
           ],
           "default": "off",
-          "description": "Traces the communication between VS Code and the UI5 language server."
+          "description": "Trace the communication between VS Code and the UI5 language server."
+        },
+        "UI5LanguageAssistant.codeAssist.deprecated": {
+          "scope": "resource",
```

Comment:
how does `"scope": "resource"` work if a user wants both global settings and per document settings?

would it be:
> "*" : true
> "foo/bar.view.xml" : false

?

Reply:
It means that it's the lowest possible level to change the setting (I don't think there is UI in vscode or theia for resource-level settings though, so it's probably the same as workspace-level settings here). The settings are managed in the client, and the server asks for the document-level settings (we cache the result in the server).  When the settings change we clear the cache so we know to ask again.

Reply:
I added some screenshots in one of the questions below